src/main/java/wolox/training/models/Book.java

```diff
@@ -4,13 +4,17 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import lombok.Data;
 import org.springframework.lang.NonNull;
 
 @Entity
+@Data
 public class Book {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "book_generator")
+    @SequenceGenerator(name = "book_generator", sequenceName = "book_seq")
     private long id;
```

Comment:
Great! And you can make the id setter private with lombok too because of what I told you before  :) you can read how to do it with Lombok [here](https://projectlombok.org/features/Data)